cherry_picker.py

```diff
@@ -5,16 +5,22 @@
 
 
 @click.command()
+@click.option('--dryrun', is_flag=True)
```

Comment:
Would you be OK with `--dry-run`? That will still give `dry_run` as the parameter name, and make the CLI spelling consistent with other tools like `git` and `behave`.